### PR TITLE
Disable hash validation in `has` method

### PR DESCRIPTION
**Pull request by SimenB**
## JIRA Issue
N/A

## Status
**READY**

## Description

This duplicates the check added in 89b6ecdab36dca79f487a3ca75ee8712dacef450.

## Todos
- [ ] Tests
- [ ] Documentation

## Related PRs
* None

### CODE REVIEW

File: lib/sink.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const JSONStream = require('JSONStream');
 const gcsClient = require('@google-cloud/storage');
 const isObject = require('lodash.isobject');
 const isString = require('lodash.isstring');
-const common = require('asset-pipe-common');
+const common = require('@asset-pipe/common');
```
Comment (digitalsadhu) [inline lib/sink.js]: oh good catch

File: lib/sink.js [hunk 2]
```diff
@@ -239,7 +239,9 @@ module.exports = class SinkGCS extends EventEmitter {
     async has(fileName) {
         assert(fileName, '"fileName" is missing');
         await this[ready];
-        const result = await this[getGcsFile](fileName).exists();
+        const result = await this[getGcsFile](fileName).exists({
+            validation: false,
```
Comment (digitalsadhu) [inline lib/sink.js]: What was the motivation for this? I've been hunting through the gcs docs but getting fed up trying to track down where they heck options is explained.

Comment (SimenB) [reply lib/sink.js]: The asset server crashed, meaning finn.no wqs without js and css

Comment (digitalsadhu) [reply lib/sink.js]: How was this able to bring down the frontpage? Only optimistic bundling was changed today and the frontpage doesnt use it. What am I missing?

Comment (digitalsadhu) [reply lib/sink.js]: oh right, the server crashed. derp

Comment (digitalsadhu) [reply lib/sink.js]: Do you have a reference to the docs for this option (or any options) Those docs are terrible? I feel like a worthless incapable human being after spending 10 mins trying to find out what the options are.

Comment (SimenB) [reply lib/sink.js]: I found it by looking through the source code the last time. I'll see if I can dig it up

**Pull request merged**